matter/wifi/wfx_host_events.h

```diff
@@ -76,95 +76,108 @@
 
 #include "sl_status.h"
 
-// wf 200 S
-#define STACK_SIZE_1K	1024
-#define TASK_PRIORITY_1    1
-#define ETH_FRAME	0
-#define CMP_SUCCESS		0
-#define COPY_6_CHAR		6 
-#define RETRY_CNT		5
-#define STATUS_0		0
-#define BITS_TO_WAIT	0
-#define DELAY_250MS		250
-#define CONNECTION_STATUS_SUCCESS	0
-#define IP_STATUS_FAIL		0
-#define INDEX 	0
-#define LENGTH_16   16
-#define GET_IPV6_SUCCESS	1
-#define GET_IPV6_FAIL	0
-#define BEACON_1	0
-#define EMPTY_BUFFER	0
-#define CHANNEL_LIST	(const uint8_t *)0
-#define CHANNEL_COUNT	0
-#define IE_DATA	(const uint8_t *)0
-#define IE_DATA_LENGTH	0
-#define BSSID_SCAN	(const uint8_t *)0
-#define	CHANNEL_0	0
-#define	PREVENT_ROAMING		1
-#define DISABLE_PMF_MODE	0
-#define STA_IP_FAIL		0
+#ifdef RS911X_WIFI
+#define WLAN_TASK_STACK_SIZE		1024
+#define WLAN_TASK_PRIORITY    		1
+#define WLAN_DRIVER_TASK_PRIORITY	1
+#define MAX_JOIN_RETRIES_COUNT		5
+
+#else  /* WF200 */
+#define WLAN_TASK_STACK_SIZE		1024
+#define WLAN_TASK_PRIORITY    		1
+#define MAX_JOIN_RETRIES_COUNT		5
+#endif
+
+// WLAN related Macros
+#define ETH_FRAME			0
+#define CMP_SUCCESS			0
+#define BSSID_MAX_STR_LEN		6 
+#define MAC_ADDRESS_FIRST_OCTET		6 
+#define AP_START_SUCCESS		0
+#define BITS_TO_WAIT			0
+#define DELAY_250MS			250
+#define CONNECTION_STATUS_SUCCESS	1
+#define IP_STATUS_FAIL			0
+#define INDEX 				0
+#define LENGTH_16   			16
```

Comment:
Should be removed

Reply:
removed below lines
#define INDEX 				0
#define LENGTH_16   			16

other 2 macros is well defined
#define CONNECTION_STATUS_SUCCESS	1
#define IP_STATUS_FAIL			0